install/scripts/craycli_init.py

```diff
@@ -0,0 +1,763 @@
+# #!/usr/bin/env python3
```

Comment:
There are multiple instances in this script where **cli** should be changed to **CLI**
Not a big deal, but they don't pay me the big bucks to find big problems! Or do they? Crap!

Also, except when referring to a pod name or similar, I think **keycloak** should be **Keycloak**